worker-batch-ingestion-container/src/test/java/com/github/cafdataprocessing/worker/ingestion/BatchWorkerAcceptanceIT.java

```diff
@@ -63,21 +62,28 @@
     public BatchWorkerAcceptanceIT() throws IOException, TimeoutException
     {
         workflow_queue = "worker-workflow";
+
         output_queue = System.getenv("CAF_BATCH_WORKER_ERROR_QUEUE");
         log.debug("Output queue: " + output_queue);
+
         final ConnectionFactory factory = new ConnectionFactory();
         factory.setHost(System.getenv("RABBITMQ_TEST_HOST"));
         factory.setPort(Integer.parseInt(System.getenv("RABBITMQ_TEST_PORT")));
-        conn = factory.newConnection();
+        factory.setUsername("guest");
+        factory.setPassword("guest");
```

Comment:
can you make access to guest here be via environment variable similar to CAF_SCHEDULER_INPUT_QUEUE